libs/blocks/countdown-timer/countdown-timer.js

```diff
@@ -0,0 +1,242 @@
+import { createTag } from '../../utils/utils.js';
+
+export class CountdownTimer extends HTMLElement {
```

Comment:
I thought Milo blocks were not supposed to use classes and web components. Normally classes are defined in `web-components` folder.
What is the reason for doing this way?